scipy/signal/tests/test_filter_design.py

```diff
@@ -3707,6 +3707,12 @@ def test_invalid_input(self):
             with pytest.raises(ValueError, match='fs must be divisible '):
                 iircomb(*args, fs=fs)
 
+        # https://github.com/scipy/scipy/issues/14043#issuecomment-1107349140
+        # Previously, fs=44100, w0=49.999 was rejected, but fs=2,
+        # w0=49.999/int(44100/2) was accepted. Now it is rejected, too.
+        with pytest.raises(ValueError, match='fs must be divisible '):
+            iircomb(w0=49.999/int(44100/2), Q=30)
```

Comment:
It would be good to test `w0 = 49.999` and `fs = 44100` as well for completeness.

Reply:
@endolith feel free to ping me once you push a commit for this and I'll merge

Reply:
@larsoner ok done